src/metabase/api/field.clj

```diff
@@ -78,6 +78,7 @@
       (create-field-values-if-needed! field))))
 
 
+;; TODO - not sure this is used anymore
```

Comment:
ok, something to re-enable then. it would effectively give us ENUM support, etc'